### ISSUE TITLE
LG AC Energy and Room temperature not updating

### ISSUE THREAD

**Issue by SN0WTR00PER** [ollo69/ha-smartthinq-sensors#210]
Hi Ollo,

There is a little issue with the room temperature and energy sensors reading on my LG Dualcool AC unit.

The integration was done without issues (Great work on this) :)

The sensors value does not update until I log on LG ThinQ App.  Bot value get the right value as soon as the value are shown on my smart phone app.

I've found a similar topic somewhere but I dont remember where it was.

I want to now what API version my unit is using cause apparently, units working with the API2 is hard to implement in HA.

Thanks in advance for your help on this  :)

SnowTrooper

**Comment by ollo69** [comment]
This is a know issue. 
To fix this, somebody with AC API2 device should capture traffic between app and LG cloud when app is opened and report it here, so that I can replicate this inside integration.

**Comment by SN0WTR00PER** [comment]
I would like to help you on this..  But how to capture traffic between the app and LG cloud?  Can you just send me few instruction where to begin.  I like to dig that kind of info myself, but I need a starting point :)

Thanks for your help!

**Comment by SN0WTR00PER** [comment]
Unfortunately, I don't Own an iOS device.

If we use an emulator to run an iOS app under windows 10... maybe it's possible to capture the traffic?

**Comment by hmoffatt** [comment]
I think a really old Android version might work too, if the LG app still supports it. The Windows simulator should work too if it is running the real OS and app in simulation.

**Comment by ollo69** [comment]
Unfortunately with Android is a Dog biting its own tail. Android app do not allow proxy from version 7 and Thinq app require minimum Android version 7 to be installed. As I know only solution is Ios.

**Comment by hmoffatt** [comment]
LG could fix their app to honour the system proxy and certificate settings, but they probably won't.

**Comment by SN0WTR00PER** [comment]
I don't know how many users of home assistant want these infos to correctly refresh every 5 minutes.  Maybe, as the community grows, our voice will reach some folks at LG dev team.

I'm going to just buy some true zigbee energy monitoring kit and get leave that aside for the moment.

If anybody find a fix for this... it could be a good finale :)

**Comment by Diegaco2005** [comment]
Hi!
I've two devices on ThinQ app, and one of this has the issue to report values to HA, while other device works well.
I own an iOS. How I can capture the traffic to help with this issue? I never used the mitmproxy.

**Comment by SN0WTR00PER** [comment]
> 
> 
> Hi! I've two devices on ThinQ app, and one of this has the issue to report values to HA, while other device works well. I own an iOS. How I can capture the traffic to help with this issue? I never used the mitmproxy.

Hi Deigaco2005.

Maybe one of your devices works under the API V2 (The one that have the issue) and the other work with the API V1 (The one that works without issue to report energy and temp values).

My LG AC works with the API V2 and randomly, it updates the values, and sometimes, it doesn't.

I hope you could help ollo with that.

I've tried everything.  Use a US account, use a CA account, with or without social media login.

The issue is there.

I think that the AC report temp and energy consumption to LG servers when there is a request from the LGThinQ app (when we log into the app or when we check the log in the app).

I think Thats why the value are not updated regularly on the server.  The data are stored (maybe) on the device until we make a request on the LG ThinQ app to check it.

To Ollo69 : Is it possible to initiate the device to send data each 5 minutes to the server, so, when HA?

Maybe I'M wrong about this, but it makes sense to me why the data is not updating.

**Comment by ollo69** [comment]
> Ok, I've installed mitmproxy on my laptop. I've found an old Motorola phone in my old stuff. I'm going to check if there is a way to run LGThinQ app on that old Jurassic Droid :)

Not possible with Android, You need an apple device

**Comment by SN0WTR00PER** [comment]
Nice 👍

**Comment by ollo69** [comment]
Just released new version with fix.

**Comment by ollo69** [comment]
There are some minor changes, but nothing important. But if you use HACS I suggest to update to last version to stay aligned.

**Comment by SN0WTR00PER** [comment]
@ollo69 

Its not a push, but a POST that is sent to LG servers by LG ThinQ app.

1 POST act as a credential (I think) that is used to identify the device, the program, the version of LG ThinQ app.  So, when the second POST is sent by the app, the post with the command (AllEventEneable) with the (airState.mon.timeout), the server knows it is legit, and then, the value are updated in Home Assistant.

If I close the app LG ThinQ, and I resend the POST with the user id   in it, and then, I resend the POST that contains the (AllEventEneable) command, the value keeps updating in Home Assistant.

And sorry for my bad english.  My first language is French.  I try very hard to make myself clear and be understood :-)

**Comment by SN0WTR00PER** [comment]
And I try to take note of the maximum time intervall between these calls. 
POST with the user id every 5 minutes
POST with the (AllEvenEneable) every 70 seconds.

I'll update you with my findings.

**Comment by rsieiro** [comment]
Hey guys! I started using this integration a couple days ago with a LG AC and I've been seeing the same behavior as @SN0WTR00PER, so I've been following this issue hoping you guys can figure out how to update those values properly :)

Not sure it helps, but just a small note: LG *might* be using that `POST` to the `/push` endpoint as an authentication, but the payload seems to be just to register the Android device to receive push notifications. `GCM` is `Google Cloud Messaging` (the service Android uses for push notifications) and the random string in `pushId` is just a token that Android generates per device when you ask for push notification permissions. So there's no authentication data embedded in that payload AFAIK, you should be able to send any random string there.

**Comment by SN0WTR00PER** [comment]
Thanks for the input.

@ollo69,  do you think you could try to code this push command so I could test it.  I had the same conclusion as rsiero.  But I dont kniw how to make this push id to work.

**Comment by SN0WTR00PER** [comment]
Is it possible that the gateway URL is not correct?  When I use mitmproxy to send the allEverEnable/airState.mon.timeout command, the url is https://aic-service.lgthinq.com:46030/v1/service/devices/....

**Comment by ollo69** [comment]
> Not sure it helps, but just a small note: LG _might_ be using that `POST` to the `/push` endpoint as an authentication, but the payload seems to be just to register the Android device to receive push notifications. `GCM` is `Google Cloud Messaging` (the service Android uses for push notifications) and the random string in `pushId` is just a token that Android generates per device when you ask for push notification permissions. So there's no authentication data embedded in that payload AFAIK, you should be able to send any random string there.

This is a good catch, but why I should send this kind of message? I cannot register to any event because this connection come from a not Android device, so probably is just unnecessary.


> Is it possible that the gateway URL is not correct?

I think is the same, but yo should enable debug in HA for integration to compare the command set by HA with the command send by your tablet

**Comment by a7x88** [comment]
Hi @ollo69 llo and @SN0WTR00PER - just to say I have just installed an LG dual cool and have noticed the same behaviour. Happy to help with any pcaps or similar if required. Thanks for the great work

**Comment by SN0WTR00PER** [comment]
> > Not sure it helps, but just a small note: LG _might_ be using that `POST` to the `/push` endpoint as an authentication, but the payload seems to be just to register the Android device to receive push notifications. `GCM` is `Google Cloud Messaging` (the service Android uses for push notifications) and the random string in `pushId` is just a token that Android generates per device when you ask for push notification permissions. So there's no authentication data embedded in that payload AFAIK, you should be able to send any random string there.
> 
> This is a good catch, but why I should send this kind of message? I cannot register to any event because this connection come from a not Android device, so probably is just unnecessary.
> 
> 
> > Is it possible that the gateway URL is not correct?
> 
> I think is the same, but yo should enable debug in HA for integration to compare the command set by HA with the command send by your tablet

@ollo69  How to activate Debug log for LGE components?

**Comment by ollo69** [comment]
Set `configuration.yaml` logger section in the following way:

```
logger:
  default: info
  logs:
    custom_components.smartthinq_sensors: debug
```

You can also set (or leave if already exist) `default: warning` if you prefer

**Comment by a7x88** [comment]
it seems my HA is sending the allEventEnable/airState.mon.timeout to the same address as my iPhone

HA
`2021-10-30 22:29:42 DEBUG (SyncWorker_1) [custom_components.smartthinq_sensors.wideq.device] Setting new state for device -snip-: allEventEnable - Set - airState.mon.timeout - 290
2021-10-30 22:29:42 DEBUG (SyncWorker_1) [custom_components.smartthinq_sensors.wideq.core_v2] lgedm2_post before: https://eic-service.lgthinq.com:46030/v1/service/devices/-snip-/control-sync
2021-10-30 22:29:42 DEBUG (SyncWorker_1) [custom_components.smartthinq_sensors.wideq.core_v2] lgedm2_post after: {'resultCode': '0000', 'result': ''}`

iPhone App is sending 
`{
    "command": "Set",
    "ctrlKey": "allEventEnable",
    "dataKey": "airState.mon.timeout",
    "dataValue": "70"
}`
to https://eic-service.lgthinq.com:46030/v1/service/devices/-snip-/control

response 
`{
    "result": "",
    "resultCode": "0000"
}`

**Comment by ollo69** [comment]
Yes, I already noted the difference in the path used for the command. I can try to change the path to `control` for this specific command and release a beta version so you could check if something will change.

**Comment by SN0WTR00PER** [comment]
> Yes, I already noted the difference in the path used for the command. I can try to change the path to `control` for this specific command and release a beta version so you could check if something will change.

@ollo69 I'll test it when youre done with the modifications.  Thanks 👍

**Comment by ollo69** [comment]
Just committed the change in `master` branch. For the moment I set the value for parameter `airState.mon.timeout` to `70`.
Please test this installing directly from `master`.

**Comment by SN0WTR00PER** [comment]
I'll give you an update.

**Comment by SN0WTR00PER** [comment]
Ok.  I can see in the debug log the traffic between HA and LG ThinQ gateway.

Its the same I can observe in mitmproxy.
No error found in the log.  Fetching data success.

I imagine its not an url issue.

As I said before, I think the server need to ask the AC unit its updated value.

It was working but I'm sure LG changed something.

2021-10-30 21:12:05 DEBUG (SyncWorker_5) [custom_components.smartthinq_sensors.wideq.device] Setting new state for device 9fe0fdee-aca7-11a4-acb5-402f86ba6a1e:  allEventEnable - Set - airState.mon.timeout - 70
2021-10-30 21:12:05 DEBUG (SyncWorker_5) [custom_components.smartthinq_sensors.wideq.core_v2] lgedm2_post before: https://aic-service.lgthinq.com:46030/v1/service/devices/9fe0fdee-aca7-11a4-acb5-402f86ba6a1e/control
2021-10-30 21:12:06 DEBUG (SyncWorker_5) [custom_components.smartthinq_sensors.wideq.core_v2] lgedm2_post after: {'resultCode': '0000', 'result': ''}
2021-10-30 21:12:06 DEBUG (SyncWorker_5) [custom_components.smartthinq_sensors.wideq.core_v2] Set V2 result: {'resultCode': '0000', 'result': ''}
2021-10-30 21:12:06 DEBUG (SyncWorker_5) [custom_components.smartthinq_sensors] ThinQ status updated
2021-10-30 21:12:06 DEBUG (MainThread) [custom_components.smartthinq_sensors] Finished fetching smartthinq_sensors-LG Dualcool data in 1.196 seconds (success: True)

**Comment by ollo69** [comment]
Sorry but I do not understand if this is solving the issue or not

**Comment by SN0WTR00PER** [comment]
Is it possible to lure the LGThinq

> 
> 
> Sorry but I do not understand if this is solving the issue or not

No.  Its not working.

**Comment by SN0WTR00PER** [comment]
I want to test if I change the client ID  info in the cove V2 file if I can make the server think that I'm using a legit LG ThinQ app, so, the allEventAirState will trigger the AC unit to send its data..

**Comment by ollo69** [comment]
> I want to test if I change the client ID info in the cove V2 file if I can make the server think that I'm using a legit LG ThinQ app, so, the allEventAirState will trigger the AC unit to send its data..

Probably the toke will be invalid after this change, but you can try. Unfortunately I do not have idea on how the help you not having a device to test, but if you find some possible solution I will be happy to implement in the code.

**Comment by SN0WTR00PER** [comment]
I'm really beginning to think to go the other way... the Shelly Power meter way.  And I want to monitor my main panelboard... So, its not a big deal to buy 2 kits instead of one.

I fear the if we find a way to make it work again, LG will probably cut the grass short under our feet again.

At the same time, If we succeed to make the whole component to work, I was going to ask if we can implement a way to display the filter timer.  A value that when it reach 360h, we need to clean the filter (a kind on reminder).  It reset after 24h and the vakue reset itself to 0.  When I read the debug log, the values stay the same, same bug that the one witb Temp and energy value...

2021 and we cant access our own ac units to monitor them as we see fit.

**Comment by a7x88** [comment]
Happy to email a complete set of HA debug and MITM logs over if it's useful.

I would plan to start HA debugs for 10 minutes, then open the Thinq app for 10 minutes. Unless something else is preferred?

**Comment by ollo69** [comment]
Sorry, i think that I close this by mistake... 😉

**Comment by ollo69** [comment]
> Happy to email a complete set of HA debug and MITM logs over if it's useful.

Any additional info will be helpful, please send to address `aolivero@gmail.com`. Let me know when you have take note of e-mail addr so that I will remove from thread to avoid spam.

It would be helpful if somebody could temporally share the LG account information (changing the pwd) so that I can execute some additional test connected with your devices, but of course this is up to you....

**Comment by a7x88** [comment]
Got it - I will aim to do the logging tonight (GMT)

Happy to change my password and share login details with you - will email over with logs tonight

**Comment by a7x88** [comment]
Sent it all over

**Comment by ollo69** [comment]
Thanks, I will start analyzing this tomorrow morning. I will try to connect with your account, but I will avoid to send command, just to check incoming data.👍

**Comment by ollo69** [comment]
I have probably find a solution using @a7x88 account's that he share for test.
The fix for the moment is committed in `master`, I will release a new version on next days after some additional checks. If you want you can start to perform some test installing directly from `master` and report here your feedback.👍

**Comment by a7x88** [comment]
Seems to be working for me 👍

**Comment by SN0WTR00PER** [comment]
It works for me too...

**Comment by ollo69** [comment]
Released new version with fix. We hope it lasts...

**Comment by MarekLe** [comment]
I have 3 LG air conditioners. Each of them is shown by the sensor that it consumes 50W. Is it normal?

**Comment by ollo69** [comment]
Did you upgrade integration to last version?

**Comment by MarekLe** [comment]
> Did you upgrade integration to last version?

Yes, I have last version. Sensor is updating now and it's great. But this problem (50W...) is from the beginning.

**Comment by ollo69** [comment]
When AC is off in the payload is always returned 50. Don't know if is correct, but is the value provided by API.

**Comment by ollo69** [comment]
I was just reading an LG user manual that, talking about display current energy usage, says: "Display wattage under 10 kW with 0.1 kW unit, and display one over 10 kW with 1 kW unit". I don't know if this apply also to data provided by API, you could do some test and check if it make sense, based on the value provided when unit is on.

**Comment by SN0WTR00PER** [comment]
It's normal that the unit use 50W when it's in standby mode.  Every component with a wifi link or ethernet communication link need power to keep its primary functions working.

Mine have the same behaviour!

And the data are ok.  My unit is rated to pull 2000W max from the breaker, so... The max current that have been registered was 1250W...  it makes sense cause the maximum load on a breaker need to be 80% of the protection.

**Comment by smackywolf** [comment]
> It's normal that the unit use 50W when it's in standby mode. Every component with a WiFi link or Ethernet link need power to keep its primary functions working.
> 
> Mine have the same behaviour!
> 
> And the data are ok. My unit is rated to pull 2000W max from the breaker, so... The max current that have been registered was 1250W...

Man, I know it says that but also I wonder how true it is. Surely they don't use 50w to standby, that's super high. I have the overall house power monitoring setup with home assistant glow, I'll do some testing and see if maybe that's a lie from the API. The power usage in the app doesn't seem to show it to be true, either, you'd expect the daily usage to be something rather than the nothing it displays.

**Comment by ollo69** [comment]
I have a dual split Mitsubishi, standby usage is 3W (measured by a separated device before the units, so this real consumption). So I agree that 50W in standby seems too high, more probably is 5W. Anyway because you have the app, instead discuss if the value is correct or not (that for sure is the value provided by API), compare the value provided by the 2 sources (HA and phone App) to find a possible rule, as explained in my previous post.

**Comment by smackywolf** [comment]
> I have a dual split Mitsubishi, standby usage is 3W (measured by a separated device before the units, so this real consumption). So I agree that 50W in standby seems too high, more probably is 5W. Anyway because you have the app, instead discuss if the value is correct or not (that for sure is the value provided by API), compare the value provided by the 2 sources (HA and phone App) to find a possible rule, as explained in my previous post.

Will do! I'll post back my findings.

**Comment by rsieiro** [comment]
I just checked here and compared with the app. The value reported by the API while the AC is off seems to be invalid (it's always 50W here too).

My AC has been off for the past 2 days, and in the app it shows zero energy consumption. I just turned it on in fan mode, energy current has increased to 66W in HA, and energy consumption in the app started increasing for today. The app shows current consumption as 0.07kW, which makes sense.

Unfortunately the kill-a-watt that I have here is 110V only and my AC is 220V, so I can't measure actual consumption in the outlet.

**Comment by ollo69** [comment]
I could just return 0 if AC is off. Probably also this is not totally true, but will not provide invalid measure calculation

**Comment by rsieiro** [comment]
Now that HA has an "Energy" tab, I was planning to investigate what is needed for this value to appear there (right now I only see energy consumption from a few smart lamps I have there, seems like HA needs the integration to provide some sort of value over time through a formula or something), would be fun to track it over time. For that scenario, I think that it should consider 0 when off.

**Comment by SN0WTR00PER** [comment]
@rsieiro 

I've already added theses lines in my configuration.yaml... It create an energy consumption sensor that can by added to the energy tab configs.


sensor:
    # LG Dualcool Energy consumption sensor (kWh)
  - platform: integration
    name: LG Dualcool Energy consumption
    source: sensor.lg_dualcool_energy_current
    unit_prefix: k
    round: 2

Here is a screenshot of the energy consumption and of course, other sinope thermostat in the lot.

![image](https://user-images.githubusercontent.com/18689208/140432694-a8a603a0-9700-4a00-b942-745f8cfaa0c3.png)

And a dashboard of the unit alone...

![image](https://user-images.githubusercontent.com/18689208/140432764-e06162aa-36cb-4007-ad15-d18e165ce8ce.png)


Hope it help a bit...

**Comment by ollo69** [comment]
> HA needs the integration to provide some sort of value over time through a formula or something

Integration cannot provide this information, because it only have information about power (W) and not energy (Wh). The platform [statistic](https://www.home-assistant.io/integrations/statistics) must be used to store power value in the recorder and calculate the used energy.

**Comment by SN0WTR00PER** [comment]
When you manually add a sensor in the energy tab section, it does exactly what it need... it use the watt consumption value, store it over a period of time, after 2 hours, it can do the math... otherwise, it's not working...The graph show 0kwh  See my screen capture from the energy tab page and the LG ThinQ app... it's working...  And i can observe the same value on the LG ThinQ app...

![image](https://user-images.githubusercontent.com/18689208/140433984-b5b63b3c-2c8a-45b8-8719-99688783a98d.png)
![Screenshot_20211104_192924_com lgeha nuts](https://user-images.githubusercontent.com/18689208

![image](https://user-images.githubusercontent.com/18689208/140434152-4d53ee88-43e2-4fc4-8c24-faa1cc460bcb.png
/140434205-743ee65b-8ca4-42c1-95d0-2ffbd60ab7ec.jpg)
![Screenshot_20211104_192943_com lgeha nuts](https://user-images.githubusercontent.com/18689208/140434248-fe0fa4cd-9e41-406f-b58a-931729d41983.jpg)
)

**Comment by SN0WTR00PER** [comment]
![image](https://user-images.githubusercontent.com/18689208/140434400-944590ed-26da-4f7d-84ed-ec98c47f3acf.png)

**Comment by SN0WTR00PER** [comment]
![image](https://user-images.githubusercontent.com/18689208/140434657-b79667d4-a304-4006-a0f5-77e88718f3cf.png)

The graph show the same behaviour in both apps...

**Comment by SN0WTR00PER** [comment]
The explanations on the home assistant website are quite clear...

https://www.home-assistant.io/integrations/integration/

**Comment by SN0WTR00PER** [comment]
Once the sensor is created in (configuration.yaml), restart Home Assistant, go to Configuration -> Energy and then, select Add single device....  And let the software to collect the data from the sensor.  Home Assistant ask to wait for 2 hours before we can see a result (data) to populate the graph...

![image](https://user-images.githubusercontent.com/18689208/140435254-bd7c218f-d62b-4889-bd11-41a551abd86d.png)

**Comment by SN0WTR00PER** [comment]
Almost forgot this... When the sensor is created, it need to be in the "Energy" class....  If not, the sensor will not be available in the "Add a device" configuration wizard....

**Comment by rsieiro** [comment]
That's awesome, thanks @SN0WTR00PER! But it's still considering the 50W usage when the AC is off, right?

**Comment by SN0WTR00PER** [comment]
I dont mind a 50 watt consumption....  But if it matters to you, ask a modified version of the lge components to ollo69 :) Maybe Ill give it a try...

**Comment by smackywolf** [comment]
So in my SUPER scientific testing of this (i did as best i could with HA Glow which is decently accurate but probably not accurate enough to really tell this) , both units (we a pair of split systems, v1 and v2) definitely do not seem to pulling 50w actual while idle. My 2.5kw unit seems to pull around 4w, and the bigger 9.4kw system around 8w. 

My suspicion is 50w is just a default value from LG, and it may not actually measure idle draw. I guess the question is: overestimate, or under estimate the value? To me, a 0 value probably makes more sense than an overestimated one. Another question is probably does this matter at all. :V

**Comment by SN0WTR00PER** [comment]
Then, a default 5 watts instead of 0 or 50 could be more acurate than nothing at all...

**Comment by MarekLe** [comment]
@ollo69 thank you for than update (5W in stand by) !!!

**Comment by marciogranzotto** [comment]
Maybe having an option to change the idle consumption would be a good idea?

**Comment by ollo69** [comment]
> Maybe having an option to change the idle consumption would be a good idea?

My opinion is that add a general option that apply only to some devices (AC v2) to tune a couple of watt is quite useless and could just confuse users.

**Comment by berendhaan** [comment]
> 
> 
> Almost forgot this... When the sensor is created, it need to be in the "Energy" class.... If not, the sensor will not be available in the "Add a device" configuration wizard....

What do you mean by this?
I've added the config as you showed but (after restart) I'm not able to select the sensor in the Energy tab.

**Comment by marciogranzotto** [comment]
> > Almost forgot this... When the sensor is created, it need to be in the "Energy" class.... If not, the sensor will not be available in the "Add a device" configuration wizard....
> 
> What do you mean by this? I've added the config as you showed but (after restart) I'm not able to select the sensor in the Energy tab.
More info: https://developers.home-assistant.io/docs/core/entity/sensor

Yeah, actually the `Energy current` sensor is of type `power` (in W), not `energy` (in Wh or kWh). That's why it won't show up on the energy tab.
Actually, it should probably be renamed to `Current Power` with `device_class: power` and `state_class: measurement`.
If you want to have it in `kWh` to show up in the energy tab, you need to create a `integration` sensor, like:
```yaml
sensor:
  - platform: integration
    source: sensor.your_ac_energy_current
    name: 'AC energy spent'
    method: left
    unit_prefix: k
    round: 2
```

**Comment by ollo69** [comment]
> Actually, it should probably be renamed to `Current Power`

This is a breaking change, after this you would reconfigure all dependencies. I would prefer to avoid this, because it will cause opening a lot of new issue. 

> with `device_class: power` and `state_class: measurement`.

This is already true

**Comment by berendhaan** [comment]
> 
> 

> 
> Actually, it should probably be renamed to `Current Power` with `device_class: power` and `state_class: measurement`. If you want to have it in `kWh` to show up in the energy tab, you need to create a `integration` sensor, like:
> 
> ```yaml
> sensor:
>   - platform: integration
>     source: sensor.your_ac_energy_current
>     name: 'AC energy spent'
>     method: left
>     unit_prefix: k
>     round: 2
> ```

This was exactly my point, I've add the config as shown by @SN0WTR00PER, which is the same as yours, but it is not selectable in the Energy Tab. Is there something wrong in the plugin connecting to the AC, or am I missing something.

**Comment by SN0WTR00PER** [comment]
I donk know why its not working for you.

If you search for the new sensor in your entities listing, do the sensor show up in the list?

If the sensor exist, you can create a Lovelace card to be sure its working... 

Maybe try to name your new sensor like this
AC energy spent instead of 'AC energy spent'

I've manage to make this work and I'm a pure newbie with Home Assistant...

😂😂

**Comment by berendhaan** [comment]
> I donk know why its not working for you.
> 
> If you search for the new sensor in your entities listing, do the sensor show up in the list?
> 
> If the sensor exist, you can create a Lovelace card to be sure its working... 
> 
> Maybe try to name your new sensor like this
> AC energy spent instead of 'AC energy spent'
> 
> I've manage to make this work and I'm a pure newbie with Home Assistant...
> 
> 😂😂

Turning the airco on did the trick 🙈 the 5W idle consumption did not trigger the sensor to be added.

**Comment by ollo69** [comment]
Closing this issue, this should be working now.
